src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -51,9 +52,6 @@
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
   <ItemGroup>
-    <BoundTreeDefinition Include="BoundTree\BoundNodes.xml">
```

Comment:
I'm not sure why it was this way before. @jaredpar did that serve some purpose?

Reply:
I can revert this change. I made it because I couldn't see the BoundTreeNodes in the solution explorer.

Reply:
I like this change (since I had the same problem) ;-) 
Just want to confirm in case there is impact.